src/lib/index.js

```diff
@@ -5,9 +5,10 @@ import SequenceViewer from './components/SequenceViewer.react';
 import NeedlePlot from './components/NeedlePlot.react';
```

Comment:
It might be better to put these in alphabetical order!

Reply:
Good catch! Done in 0433d1f2a6e9c05e9d763d0.